src/app/users/users-profiles/users-profiles.component.html

```diff
@@ -0,0 +1,99 @@
+<div class="main-container">
+    <div class="columns-1">
```

Comment:
Pentru coloana din stanga avem nevoie de altceva. 
E un meniu care nu va lua date de la user. Aici vom afisa doar o lista de pagini pe care userul le poate accesa.
Ar merge div in loc de input pentru a afisa meniul. 
Probabil ar simplifica si codul.

Reply:
Done